src/index.html

```diff
@@ -6,9 +6,28 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#"><img src="./images/logo.png" alt="logo Moyo"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li><a href="#" class="is-active">Apple</a></li>
```

Comment:
```suggestion
          <li class="nav__item"><a href="#" class="is-active">Apple</a></li>
```
Get used to styling all elements using classes, it's more reader-friendly and you'll avoid unnecessary errors that can occur due to styling with tag selectors